cfpb/cf-core
# Updating web type spacing and leading

Updating web type spacing and leading
## Changes
- Updated text leading (line-height) to match recommendation from DM issue #349
- Updated text top and bottom margins to match recommendation from DM issue #269
## Fixes
- Fixed subheader font-weight
## Testing
- Check the demo page at `/demo/index.html` and scroll to heading section
- Install to your project with `bower install git@github.com:jimmynotjim/cf-core.git#d9f0266` _if you do this, you'll have to uninstall it and then install the regular cf-core when done testing_
## Review
- @duelj
- @nataliafitzgerald
- @Scotchester 
- FEWDs
## Screenshots

<img width="605" alt="screen shot 2015-10-26 at 5 47 57 pm" src="https://cloud.githubusercontent.com/assets/1280430/10771693/8c6e22ca-7cc8-11e5-99ec-60728dd5c97c.png">

<img width="608" alt="screen shot 2015-10-26 at 4 34 11 pm" src="https://cloud.githubusercontent.com/assets/1280430/10771694/8f3b6292-7cc8-11e5-93f3-6c4739e9de04.png">
## Notes
- We are no longer optically adjusting margins/padding to match design mockups pixel perfectly
## Checklist
